src/components/Workouts/Workouts.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FunctionComponent, useEffect, useState } from 'react';
+import Workout from '../../models/Workout';
+import Checkins from '../Checkins';
+import { fetchWorkouts } from './api';
+import WorkoutCard from './WorkoutCard';
+
+interface WorkoutsExpecting {
+    workouts: Workout[];
+}
+
+const enhancer = (Component: FunctionComponent<WorkoutsExpecting>) => () => {
+    const [workouts, loadWorkouts] = useState<Workout[]>([]);
+    useEffect(() => {
```

Comment:
I see some duplication between books, checkins, and workouts. Do you think it's possible to DRY up some of the boilerplate?

Reply:
Completely agree about this. I will try this in a new PR.